cache/src/main/java/io/envoyproxy/controlplane/cache/NodeGroup.java

```diff
@@ -14,5 +14,12 @@
    *
    * @param node identifier for the envoy instance that is requesting config
    */
-  T hash(Node node);
+  T hashV2(Node node);
```

Comment:
I'd rather use method overloading than version suffixes, so have `hash(Node)` and `hash(v3.Node)` instead of `hashV2(Node)` and `hashV3(v3.Node)`.